core/gesture.ts

```diff
@@ -743,6 +746,8 @@ export class Gesture {
     e.preventDefault();
     e.stopPropagation();
 
+    keyboardNavigationController.setIsActive(false);
```

Comment:
Out of curiosity, what happens if someone tries using the menu key? Does that simulate a right click or require a key event handler? It doesn't currently work in Blockly, and I'm unsure of the expectations for that key for accessibility (or if those expectations even matter since we have so much custom key binding, anyway).